src/lib/form-field/form-field.html

```diff
@@ -1,28 +1,17 @@
-<div class="mat-input-wrapper mat-form-field-wrapper">
-  <div class="mat-input-flex mat-form-field-flex" #connectionContainer
+<div class="mat-form-field-wrapper">
+  <div class="mat-form-field-flex" #connectionContainer
        (click)="_control.onContainerClick && _control.onContainerClick($event)">
-    <div class="mat-input-prefix mat-form-field-prefix" *ngIf="_prefixChildren.length">
+    <div class="mat-form-field-prefix" *ngIf="_prefixChildren.length">
       <ng-content select="[matPrefix]"></ng-content>
     </div>
-
-    <div class="mat-input-infix mat-form-field-infix" #inputContainer>
+F
```

Comment:
Extra `F` character here. It was very strange seeing so many screenshot failures where the inputs had an extra `F` at the front.